assets/src/edit-story/app/media/media3p/api/test/apiFetcher.js

```diff
@@ -55,16 +55,33 @@ const VALID_CATEGORIES_RESPONSE = {
   json: () => CATEGORIES_BODY_JSON,
 };
 
+const VALID_REGISTER_USAGE_RESPONSE = {
+  ok: true,
+  status: 200,
+  json: () => {},
+};
+
 const ERROR_RESPONSE = {
   ok: false,
   status: 400,
   statusText: 'Error',
 };
 
-function mockFetch(response) {
-  jest
-    .spyOn(global, 'fetch')
-    .mockImplementation(() => Promise.resolve(response));
+const PAYLOAD = '02647749feef0d5536c92df1d9cfa38e';
```

Comment:
The name 'PAYLOAD' isn't very specific, suggest 'REGISTER_USAGE_PAYLOAD'.

Reply:
Done.